ui/app/scripts/controllers/blockersCtrl.js

```diff
@@ -87,6 +91,7 @@ angular.module('apollo')
           var serviceId = null;
           var environmentId = null;
           var stackId = null;
+          var availability = null;
```

Comment:
Make sure that null is acceptable in the backend side

Reply:
Changed it directly to "var availability = $scope.blockerAvailability;" like the other String variables